www-apps/icingaweb2-module-graphite/icingaweb2-module-graphite-1.2.1.ebuild

```diff
@@ -0,0 +1,26 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+DESCRIPTION="Icinga Web 2 plugin for Graphite"
+HOMEPAGE="https://www.icinga.com/docs/graphite/latest/"
+if [[ "${PV}" == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/Icinga/icingaweb2-module-graphite.git"
+else
+	KEYWORDS="amd64 x86"
```

Comment:
This looks mostly good, but the keywords need to be `~amd64 ~x86` for a version bump (this commit needs to be rewritten)

Reply:
Yes, forgot to drop to testing, pushed an update, thanks.